From alan@linux.intel.com  Thu Aug 27 14:23:47 2009
From: Alan Cox <alan@linux.intel.com>
Date: Thu, 27 Aug 2009 10:58:59 +0100
Subject: Staging: et131x: MPSend macros
To: greg@kroah.com
Message-ID: <20090827095854.23892.40259.stgit@localhost.localdomain>


Most are unused, one is used and can be replaced with the definition

Signed-off-by: Alan Cox <alan@linux.intel.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@suse.de>
---

 drivers/staging/et131x/et1310_tx.c      |    8 ++++----
 drivers/staging/et131x/et131x_adapter.h |    5 -----
 2 files changed, 4 insertions(+), 9 deletions(-)


--- a/drivers/staging/et131x/et1310_tx.c
+++ b/drivers/staging/et131x/et1310_tx.c
@@ -1375,7 +1375,7 @@ void et131x_free_busy_send_packets(struc
 		DBG_VERBOSE(et131x_dbginfo, "pMpTcb = 0x%p\n", pMpTcb);
 
 		FreeCounter++;
-		MP_FREE_SEND_PACKET_FUN(etdev, pMpTcb);
+		et131x_free_send_packet(etdev, pMpTcb);
 
 		spin_lock_irqsave(&etdev->TCBSendQLock, flags);
 
@@ -1384,7 +1384,7 @@ void et131x_free_busy_send_packets(struc
 
 	if (FreeCounter == NUM_TCB) {
 		DBG_ERROR(et131x_dbginfo,
-		    "MpFreeBusySendPackets exitted loop for a bad reason\n");
+		    "MpFreeBusySendPackets exited loop for a bad reason\n");
 		BUG();
 	}
 
@@ -1451,7 +1451,7 @@ static void et131x_update_tcb_list(struc
 			etdev->TxRing.CurrSendTail = NULL;
 
 		spin_unlock_irqrestore(&etdev->TCBSendQLock, flags);
-		MP_FREE_SEND_PACKET_FUN(etdev, pMpTcb);
+		et131x_free_send_packet(etdev, pMpTcb);
 		spin_lock_irqsave(&etdev->TCBSendQLock, flags);
 
 		/* Goto the next packet */
@@ -1466,7 +1466,7 @@ static void et131x_update_tcb_list(struc
 			etdev->TxRing.CurrSendTail = NULL;
 
 		spin_unlock_irqrestore(&etdev->TCBSendQLock, flags);
-		MP_FREE_SEND_PACKET_FUN(etdev, pMpTcb);
+		et131x_free_send_packet(etdev, pMpTcb);
 		spin_lock_irqsave(&etdev->TCBSendQLock, flags);
 
 		/* Goto the next packet */
--- a/drivers/staging/et131x/et131x_adapter.h
+++ b/drivers/staging/et131x/et131x_adapter.h
@@ -333,9 +333,4 @@ struct et131x_adapter {
 	struct net_device_stats net_stats_prev;
 };
 
-#define MPSendPacketsHandler  MPSendPackets
-#define MP_FREE_SEND_PACKET_FUN(Adapter, pMpTcb) \
-	et131x_free_send_packet(Adapter, pMpTcb)
-#define MpSendPacketFun(Adapter, Packet) MpSendPacket(Adapter, Packet)
-
 #endif /* __ET131X_ADAPTER_H__ */
